docs/billing/external/techspec_ebs.md

```
@@ -1092,3 +1094,4 @@ ValueType="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-x509-token-pr
 
 Catalogue # CIB-XXXXXXX Month/Year © Queen’s Printer for Ontario
 
+```
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)